lib/fathead/numpy/parse.py

```diff
@@ -351,13 +351,12 @@ def get_redirects(title):
     redirects = set()
     title_split = title.split(".")
     if title_split[0] == PACKAGE_NAME:
-        redirects.add(PACKAGE_NAME + " " + ".".join(title_split[1:]))
-    else:
-        redirects.add(PACKAGE_NAME + "." + title)
+        title_split = title_split[1:]
+
+    redirects.add(".".join(title_split))
```

Comment:
remember the period is dropped by the query. We don't need both lines, just one will do! 😄